lib/express_validator.js

```diff
@@ -1,262 +1,335 @@
-/*
- * This binds the node-validator library to the req object so that
- * the validation / sanitization methods can be called on parameter
- * names rather than the actual strings.
- *
- *
- * 1. To validate parameters, use `req.check(param_name, [err_message])`
- *        e.g. req.check('param1').len(1, 6).isInt();
- *        e.g. req.checkHeader('referer').contains('mydomain.com');
+var validator = require('validator');
+
+// validators and sanitizers not prefixed with is/to
+var additonalValidators = ['contains', 'equals', 'matches'];
```

Comment:
"additional" is misspelled in `additonalValidators` and `additonalSanitizers`. Be sure to correct this variable in all places it occurs.